lib/mixlib/install/backend/package_router.rb

```diff
@@ -195,8 +197,18 @@ def map_properties(properties)
         # Generates a chef standard download uri in the form of
         # http://endpoint/files/:channel/:project/:version/:platform/:platform_version/:file
         def generate_chef_standard_path(channel, project, version, platform, platform_version, filename)
+          # For some older platform & platform_version combinations we need to
+          # use COMPAT_DOWNLOAD_URL_ENDPOINT since these versions have an
+          # OpenSSL version that can not verify the ENDPOINT based urls
+          base_url = case "#{platform}-#{platform_version}"
+                     when "freebsd-9", "el-5", "solaris2-5.9", "solaris2-5.10"
+                       COMPAT_DOWNLOAD_URL_ENDPOINT
+                     else
+                       endpoint
+                     end
```

Comment:
This logic has to go in `#endpoint` like it did previously in the `Mixlib::Install::Backend::Bintray` or API calls on these legacy platforms will also fail (since `#get` uses `#endpoint` when making the actual API requests).